src/reducer.js

```diff
@@ -36,9 +36,35 @@ const userProfile = (state = userProfileDefaultState, action) => {
   }
 };
 
+// This needs to be inside userProfile
+const userProfileDefaultState = (state = {
```

Comment:
userProfileDefaultState should be the default state you're setting inside the first argument, and the name of this method should stay userProfile. 
Be careful how you name things, it will help other people understand our code